yal/wa_fb_crossover_feedback.py

```diff
@@ -0,0 +1,257 @@
+import logging
+
+from vaccine.base_application import BaseApplication
+from vaccine.states import Choice, FreeText, WhatsAppButtonState, WhatsAppListState
+from vaccine.utils import get_display_choices
+from yal import rapidpro, utils
+from yal.askaquestion import Application as AskaQuestionApplication
+from yal.pleasecallme import Application as PleaseCallMeApplication
+from yal.utils import get_generic_error
+
+logger = logging.getLogger(__name__)
+
+
+class Application(BaseApplication):
+    START_STATE = "state_wa_fb_crossover_feedback"
+
```

Comment:
Please add another state here at the start that is a carbon copy of this one (but the name mustn't clash and it must take the user to your start state)
https://github.com/praekeltfoundation/vaccine-eligibility/blob/main/yal/content_feedback_survey.py#L11
It's needed to let RapidPro know that the user has been offered this survey so it doesn't get sent again